docs/source/cookbooks/reproducer.md

```diff
@@ -0,0 +1,129 @@
+recover anything after the run: virtual machines are destroyed and undefined,
+and the image layers are removed.
+
+## Prerequisites
+### CRC
+The CRC VM has to be properly set up prior running the reproducer play. You
+may get to that leveraging wither the rhol_crc role from the CI Framework, or
+manually by calling `crc setup && crc start -p pull-secret`.
+
+Once the CRC VM is properly started, you can then stop it and undefine it as
+follow:
+```Bash
+$ virsh destroy crc
+$ virsh undefine crc
+```
+
+As an extra step, you may also clean the existing networks:
+```Bash
+$ virsh net-list --all
+$ virsh net-destroy <NETWORK>
+$ virsh net-undefine <NETWORK>
+```
+
+You then have to ensure you have the following content on the target hypervisor:
+- ~/.crc/machine/crc/crc.qcow2
+- ~/.crc/machine/crc/id_*
+- ~/.crc/machine/crc/kubeconfig
+
+### Ssh authorized_keys
+On the target hypervisor, you have to ensure the ~/.ssh/authorized_keys has all
+of the needed public keys. It will then be copied in the various virtual machines
+the reproducer will create.
+
+## Needed parameters
+While we provide the "simple" layout, consuming three nodes, you may want
+to provide your own layout, maybe with more compute, or some other kind of
+node.
+
+In any cases, the following data are the bare minimal:
+
+- `cifmw_path`: you can take the same value as in the provided file, or extend it as needed
+- `cifmw_libvirt_manager_configuration`: this dict is mandatory in order to create the layout.
+
+In the `cifmw_libvirt_manager_configuration`, there are also some conventions
+and mandatory content/information.
+
+In the `vms` section, please ensure you have at least "controller" and "crc".
+Those names are fixed and used later as static reference.
+
+In the `networks` section, you can define your own custom networks. Usually,
+you can take the same values we provide in the existing layout.
+
+Note that, if you just want to get more compute, you can play with the `amount`
+value. It will then create `compute-[0..n]` virtual machines.
+
+### Extra parameter: cifmw_reproducer_repositories
+
+This parameter allows you to pass a list of repositories you want to sync from
+your local laptop onto the ansible controller machine. The form is pretty easy:
+```YAML
+cifmw_reproducer_repositories:
+  - src: "{{ playbook_dir }}"
+    dest: "src/github.com/openstack-k8s-operators/"
+  - src: "{{ cifmw_install_yamls_repo }}"
+    dest: "src/github.com/openstack-k8s-operators/"
+```
+This one will ensure you have ci_framework as well as install_yamls in a
+known location on the virtual machine. The dest path matches the one we usually
+get in CI.
+
+## Inventory
+This feature can be launched against your own desktop or laptop, but also
+against a remote hypervisor (preferred due to resources). You can therefore
+create your own inventory as follow:
+```YAML
+all:
+  hosts:
+    hypervisor:
+     ansible_user: virtuser
```

Comment:
zuul ?

Reply:
"whatever" - that's on the hypervisor. I don't have "zuul" on the hypervisor, for instance. That's an example.

Reply:
ansible_host: 127.0.0.1
          
As an example ?
Ir keeping there ansible_host:  <You (remote) machines address>

Reply:
(will update with some more explicit placeholder)